examples/stateful-counter/README.md

```diff
@@ -0,0 +1,222 @@
+## Example
```

Comment:
Please rename the heading to something more meaningful.

Reply:
Could you please suggest something ?